ext/project_data.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-project: 'puppet'
+project: 'puppet-bolt'
```

Comment:
i'm confused. why are we renaming the project to puppet-bolt?

Also @lutter could you open a ticket for this change? Though a small change-set it seems non-trivial enough to warrant one.